lib/tunnel.js

```diff
@@ -0,0 +1,36 @@
+const localtunnel = require('localtunnel')
+const { spawn } = require('child_process')
+const { exitOnError } = require('./common')
+
+async function start ({ port, subdomain }) {
+  console.log('tunnel start', port, subdomain)
+  try {
+    const tunnel = await localtunnel({ port, subdomain })
+
+    console.log(`Opening tunnel on ${tunnel.url}`)
+
+    // tunnel.on('request', (info) => console.log(info))
+    tunnel.on('error', (error) => console.log(error))
+
+    const ls = spawn('npm', ['run', 'dev'], {
```

Comment:
why are we trying to start the user's service from the cli?

What if they don't have a `dev` script?